conjur/api/api.py

```diff
@@ -113,7 +113,7 @@ def authenticate(self) -> str:
         """
         if not self.login_id or not self.api_key:
             # TODO: Use custom error
-            raise RuntimeError("Missing parameters in authentication invocation!")
+            raise MissingParametersException("Missing parameters in authentication invocation!")
```

Comment:
I know you didn't add the message but can you remove the `!` at the end? It feels like we are screaming at the user :P

Reply:
removed